src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -85,6 +86,14 @@ const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ åpenBehand
     const visSubmitKnapp =
         !vurderErLesevisning() && åpenBehandling?.status === BehandlingStatus.UTREDES;
 
+    const [visFeilutbetaltValuta, settVisFeilutbetaltValuta] = React.useState(false);
+    const [erUlagretNyFeilutbetaltValutaPeriode, settErUlagretNyFeilutbetaltValutaPeriode] =
+        React.useState(false);
+
+    React.useEffect(() => {
+        settVisFeilutbetaltValuta(åpenBehandling.feilutbetaltValuta.length > 0);
+    }, [åpenBehandling]);
```

Comment:
UseEffect som er avhengig av propendring https://beta.reactjs.org/learn/you-might-not-need-an-effect#adjusting-some-state-when-a-prop-changes